packages/ui/src/components/editor/ImageLink/index.tsx

```diff
@@ -23,14 +23,17 @@ All portions of the code written by the Infinite Reality Engine team are Copyrig
 Infinite Reality Engine. All Rights Reserved.
 */
 
+import { ItemTypes } from '@ir-engine/editor/src/constants/AssetTypes'
+import { ImageFileTypes } from '@ir-engine/engine/src/assets/constants/fileTypes'
 import React, { ImgHTMLAttributes, useEffect, useRef } from 'react'
 import { useTranslation } from 'react-i18next'
 import { twMerge } from 'tailwind-merge'
+import FileBrowserInput from '../input/FileBrowser'
 import ImageUrlFallback from './image-url-fallback.png'
 
 export interface ImageLinkProps extends Omit<ImgHTMLAttributes<HTMLImageElement>, 'onChange' | 'onBlur'> {
   variant?: 'lg' | 'md' | 'sm' | 'full'
-  onChange?: (value: string) => void
+  onRelease?: (value: string) => void
```

Comment:
we could keep the `onChange` prop name and pass it to FileBrowserInput
`onRelease` is more associated with sliders where we _release_ the input invoking an action